docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md

```diff
@@ -2,7 +2,7 @@
 
 This is a guide for migrating from Play Slick v0.8 to v1.0, v1.1 or v2.0.0.
```

Comment:
If this is a guide for migrating to v2.0.0, why do we mention "that you have read the [Slick 3.1 documentation](http://slick.typesafe.com/docs/)" later?

Reply:
Bad find and replace, maybe? 🤷‍♂